projects/pastanaga-angular/src/lib/styles/theme/_tables.scss

```diff
@@ -1,7 +1,10 @@
 @import "palette";
+@import "typography";
 
 $color-text-table-header:               $color-neutral-secondary-default;
```

Comment:
all `!default` are missing in this file :/ (not you, but would be good to add them)